samples/microsoft-graph-toolkit/mgt-teams-tab/.gitignore

```diff
@@ -0,0 +1,24 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Could you please rename this to tab-graph-toolkit?

Reply:
Also, please correct the folder staructure as well. Final path should look samples/tab-graph-toolkit/node

Reply:
@Trinetra-MSFT - I see that folder staructure is not changed to **/samples/tab-graph-toolkit/nodejs/mgt-teams-tab** , could you please remove **mgt-teams-tab** and move those files directly to root of **nodejs** folder?